deploy/charts/library/templates/_cluster-role.tpl

```diff
@@ -148,4 +148,15 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
     verbs: ["get", "update", "delete", "list"]
+---
+# Aspects of ceph osd purge job that require access to the cluster namespace
```

Comment:
The osd purge job uses `serviceAccountName: rook-ceph-purge-osd`, so it should not use this default service account.

Reply:
yes that comment is added by mistake